src/screen/GameScreen.java

```diff
@@ -221,19 +239,20 @@ protected final void update() {
             //	this.enemyShipSpecial = null;
             //		this.logger.info("The special ship has escaped");
             //}
-
             this.ship.update();
             this.enemyShipSet.update();
             // 1초마다 levelTime 1씩 증가
             if (this.clockCooldown.checkFinished()) {
                 this.levelTime += 1;
                 this.clockCooldown.reset();
             }
+
         }
 
         manageCollisions();
         cleanBullets();
         draw();
```

Comment:
Quit를 하면 `GameScreen`이 잠깐 보이다 `TitleScreen`으로 넘어가는 버그가 있습니다.

문제는 Quit로 인해 `GameScreen`의 `isRunning`이 `false`가 되는 것과 `GameScreen`이 종료되는 것 사이에 `draw()`가 있기 때문인 것 같습니다.

따라서 다음과 같이 `draw()` 와 같은 작업들은 `isRunning` 중일 때만 동작하도록 해보았더니 해당 버그가 해결되었습니다.
(draw를 update 메소드의 상단으로 옮기는 방법도 생각해봤지만, 혹시 모를 오류 방지를 위해 순서를 변경하진 않았습니다.)
```suggestion
        // Quit 시에(!isRunning) GameScreen 그려지지 않도록 함
        if (isRunning) {
            manageCollisions();
            cleanBullets();
            draw();
        }
```

Reply:
반영했습니다!